packages/SymbioticController/src/Adapter/Nette/Application/PresenterHelper.php

```diff
@@ -0,0 +1,25 @@
+<?php declare(strict_types=1);
+
+namespace Symplify\SymbioticController\Adapter\Nette\Application;
+
+use Nette\Application\UI\Presenter;
+
+final class PresenterHelper extends Presenter
+{
+    public function __construct()
+    {
+        parent::__construct();
+        $this->saveGlobalState();
```

Comment:
Now I recall how WTF that was fro me, thanks for reminder :) I'll improve the method description.

Reported here: https://github.com/nette/application/pull/179